lua/deferred-clipboard.lua

```diff
@@ -71,7 +66,7 @@ function M.setup(options)
     schedule_clipboard_sync_on_focus_change()
 
     if is_continuous_clipboard_sync_enabled() then
```

Comment:
After the changes I've suggested above we could use use them like this:

```suggestion
    if is_continuous_clipboard_sync_enabled() and are_focus_change_events_supported() then
```